variants/BLACK_F407VE/variant.h

```diff
@@ -0,0 +1,212 @@
+
+enum {
+  // Left Side
+  //Ext   //Int
+  //5V    //5V
+  //5V    //5V
+  //3V3   //3V3
+  //3V3   //3V3
+  //GND   //GND
+  PE2,    PE3,    // D0, D1
+  PE4,    PE5,    // PE_4: BUT K0, PE_5: BUT K1
+  PE6,    PC13,
+  PC0,    PC1,
+  PC2,    PC3,
+  //VREF- //VREF+
+  PA0,    PA1,    // D10, D11 PA_0(WK_UP): BUT K_UP)
+  PA2,    PA3,
+  PA4,    PA5,
+  PA6,    PA7,    // PA_6: LED D2, PA_7: LED D3 (active LOW)
+  PC4,    PC5,
+  PB0,    PB1,    // D20, D21
+  PE7,    PE8,
+  PE9,    PE10,
+  PE11,   PE12,
+  PE13,   PE14,
+  PE15,   PB10,   // D30, D31
+  PB11,   PB12,
+  PB13,   PB14,
+  // Right Side
+  //Int   //Ext
+  //3V3   //3V3
+  //3V3   //3V3
+  //BOOT0 //BOOT1
+  //GND   //GND
+  //GND   //GND
+  PE1,    PE0,
+  PB9,    PB8,
+  PB7,    PB6,    // D40, D41
+  PB5,    PB3,
+  PD7,    PD6,
+  PD5,    PD4,
+  PD3,    PD2,
+  PD1,    PD0,    // D50, D51
+  PC12,   PC11,
+  PC10,   PA15,
+  PA12,   PA11,   // PA_11: USB_DM, PA_12: USB_DP
+  PA10,   PA9,
+  PA8,    PC9,    // D60, D61
+  PC8,    PC7,
+  PC6,    PD15,
+  PD14,   PD13,
+  PD12,   PD11,
+  PD10,   PD9,    // D70, D71
+  PD8,    PB15,
+  PB4,
+  // Analog pins
+  PA0_A,          // D75
+  PA1_A,
+  PA2_A,
+  PA3_A,
+  PA4_A,
+  PA5_A,          // D80
+  PB0_A,
+  PB1_A,
+  PC0_A,
+  PC1_A,
+  PC2_A,
+  PC3_A,
+  PC4_A,
+  PC5_A,
+  PEND
+};
+
+// This must be a literal with the same value as PEND
+#define NUM_DIGITAL_PINS        89
+// This must be a literal with a value less than or equal to MAX_ANALOG_INPUTS
+#define NUM_ANALOG_INPUTS       14
+#define NUM_ANALOG_FIRST        75
+
+// Below ADC, DAC and PWM definitions already done in the core
+// Could be redefined here if needed
+// ADC resolution is 12bits
+//#define ADC_RESOLUTION          12
+//#define DACC_RESOLUTION         12
+
+// PWM resolution
+//#define PWM_RESOLUTION          8
+//#define PWM_FREQUENCY           1000
+//#define PWM_MAX_DUTY_CYCLE      255
+
+// On-board LED pin number
+#define LED_D2                  PA6
+#define LED_D3                  PA7
+#define LED_BUILTIN             LED_D2
+#define LED_GREEN               LED_D2
+
+// On-board user button
+#define BTN_K_UP                PA0
+#define BTN_K0                  PE4
+#define BTN_K1                  PE5
+#define USER_BTN                BTN_K0
+
+// Below SPI and I2C definitions already done in the core
+// Could be redefined here if differs from the default one
+// SPI Definitions
+#define PIN_SPI_SS              PB7 // NRF24 connector
+#define PIN_SPI_SS1             PB0 // W25Q16 (on board flash)
+#define PIN_SPI_MOSI            PB5 // NRF24 connector & W25Q16 (on board flash)
+#define PIN_SPI_MISO            PB4 // NRF24 connector & W25Q16 (on board flash)
+#define PIN_SPI_SCK             PB3 // NRF24 connector & W25Q16 (on board flash)
+
+// I2C Definitions
+//#define PIN_WIRE_SDA            14 // Default for Arduino connector compatibility
+//#define PIN_WIRE_SCL            15 // Default for Arduino connector compatibility
```

Comment:
#define PIN_WIRE_SCL            PB8

Reply:
Hello @fpistm: thank you for your update, do you still need me to post a commit for it?

Reply:
Hi @edogaldo,
I will push my branch as it is rebased on top of the master.
I've resolved all merge conflicts.
I've raised here to track this issue.